platform-service/src/unstract/platform_service/helper/cost_calculation.py

```diff
@@ -21,16 +24,40 @@ def __init__(
         self.ttl_days = ttl_days
         self.url = url
         self.file_path = file_path
+
+        if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
+            self.file_storage, self.file_path = self.__get_storage_crdentials()
         self.model_token_data = self._get_model_token_data()
 
+    def __get_storage_crdentials(self) -> tuple[PermanentFileStorage, str]:
```

Comment:
```suggestion
    def __get_storage_credentials(self) -> tuple[PermanentFileStorage, str]:
```
NIT: Typo fixed

Reply:
@gaya3-zipstack can you fix this typo as well?

Reply:
Done